pages/index.tsx

```diff
@@ -3,28 +3,42 @@ import {serverSideTranslations} from "next-i18next/serverSideTranslations";
 import {withTranslation, WithTranslation} from "next-i18next";
 import Component from "../components/application/Component";
 import {NextPageContext} from "next";
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
No need to also import bootstrap here after importing it in the _app.tsx.

Reply:
Good :D